README.md

```diff
@@ -1,10 +1,92 @@
-# Mailsac Self-Hosted Temporary Email
+# Mailsac Self-Hosted Temporary Email User Interface
 
 This project is a user interface for disposable email. It uses mailsac.com as the backend mail service.
 
+![Checking mail](images/example_animation.gif)
+
+## Use Cases
+
+[Mailsac](https://mailsac.com) already offers disposable email without a need to sign up for an account. What need does
+this application meet that Mailsac doesn't already provide?
+
+Mailsac has limitations on what can be viewed without signing up for an account. Only the latest email in a public
+mailbox can be viewed withing an account. Mail in a private domain cannot be viewed without signing in with an account
+that has permissions to the private domain.
+
+### Class Room Use Case
+
+An instructor may want students, who are young in age and don't have an email address, to sign up for an account with
+a web service used in the class. The Mailsac Self-Hosted Temporary Email User Interface application provides a
+simplified interface for students to view email sent to a private mailsac hosted domain without the need to sign up for
+a mailsac account or email address.
```

Comment:
I think just as important their users won’t have all the features of Mailsac UI, just view mail.